### PR TITLE
VADC-378 Integrate the virtual tour into the new GWAS

### PR DESCRIPTION
**Pull request by vzpgb**
Jira Ticket: [VADC-378](https://ctds-planx.atlassian.net/browse/VADC-378)

### New Features
* Integrated the virtual tour into the new GWAS app
* Added new TourButton component
* Removed unused data-tour selectors

### Screenshots
image
image
image
image

[VADC-378]: https://ctds-planx.atlassian.net/browse/VADC-378?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Analysis/GWASV2/Steps/SelectCovariates/SelectCovariates.jsx [hunk 1]
```diff
@@ -27,6 +31,10 @@ const SelectCovariates = ({
                 outcome={outcome}
                 handleClose={() => {
                   setSelectionMode('');
+                  dispatch({
+                    type: ACTIONS.SET_SELECTION_MODE,
+                    payload: '',
+                  });
```
[the remark this answers is not in the document]
Comment (pieterlukasse) [reply src/Analysis/GWASV2/Steps/SelectCovariates/SelectCovariates.jsx:37]: I tried this and it seemed to work fine:

Screenshot 2023-01-20 at 22 08 04

**Review by pieterlukasse — COMMENTED** [review]
thanks for the great work @jarvisraymond-uchicago and @vzpgb 👍 

I added some comments regarding one part that I think can be simplified. Please take a look.

File: src/Analysis/GWASV2/Steps/SelectCovariates/SelectCovariates.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (vzpgb) [reply src/Analysis/GWASV2/Steps/SelectCovariates/SelectCovariates.jsx]: Thank you for the suggestion! I applied it in the commit [e84cb4f](https://github.com/uc-cdis/data-portal/pull/1207/commits/e84cb4f1a494825d5c39d93e8f9f00ab65d917b0)

File: src/Analysis/GWASV2/Steps/SelectOutcome/SelectOutcome.jsx [hunk 2]
```diff
@@ -24,6 +28,10 @@ const SelectOutcome = ({
             outcome={outcome}
             handleClose={() => {
               setSelectionMode('');
+              dispatch({
+                type: ACTIONS.SET_SELECTION_MODE,
+                payload: '',
+              });
```
[the remark this answers is not in the document]
Comment (vzpgb) [reply src/Analysis/GWASV2/Steps/SelectOutcome/SelectOutcome.jsx]: Suggestion applied in the commit [e84cb4f](https://github.com/uc-cdis/data-portal/pull/1207/commits/e84cb4f1a494825d5c39d93e8f9f00ab65d917b0), thank you!

File: src/Analysis/GWASV2/Steps/SelectCovariates/SelectCovariates.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (vzpgb) [reply src/Analysis/GWASV2/Steps/SelectCovariates/SelectCovariates.jsx:37]: Hmm, after more testing, it looks like this change does not always work as expected. I reverted the PR to the previous commit [d73b491](https://github.com/uc-cdis/data-portal/pull/1207/commits/d73b491b5459e8d40c4be53afd4af8af17eb6319) to restore the functionality for now.

File: src/Analysis/GWASV2/Steps/SelectOutcome/SelectOutcome.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (vzpgb) [reply src/Analysis/GWASV2/Steps/SelectOutcome/SelectOutcome.jsx:34]: We found some scenarios, that do not work as expected, reverted the PR to the previous commit [d73b491](https://github.com/uc-cdis/data-portal/pull/1207/commits/d73b491b5459e8d40c4be53afd4af8af17eb6319) to restore the functionality for now.

**Review by pieterlukasse — APPROVED** [review]

**Pull request merged**